src/pages/rsvp.js

```diff
@@ -71,9 +75,11 @@ const Rsvp = {
     if (formIsValid) {
       fetch(url, {
         method: "post",
+        credentials: "include",
         headers: {
           "Accept": "application/json, text/plain, */*",
           "Content-Type": "application/json",
+          "X-CSRFToken": csrfToken,
```

Comment:
I feel like we should test empty string, null, made-up token, and expired token / other session's token here.